spot_rl_experiments/spot_rl/utils/img_publishers.py

```diff
@@ -405,12 +405,13 @@ def inference(self, hand_rgb, timestamp, stopwatch):
     parser.add_argument(
         "--local", action="store_true", help="fully local robot connection"
     )
-    parser.add_argument(
-        "--bounding_box_detector",
-        choices=["owlvit", "mrcnn"],
-        default="owlvit",
-        help="bounding box detector model to use (owlvit or maskrcnn)",
-    )
+
+    # parser.add_argument(
+    #     "--bounding_box_detector",
+    #     choices=["owlvit", "mrcnn"],
+    #     default="mrcnn", #"owlvit" # we need to change this behaviour, take the model name from config.yaml, otherwise it defaults to OwlVIT
+    #     help="bounding box detector model to use (owlvit or maskrcnn)",
+    # )
```

Comment:
Why are we commenting this block? This will have implications in ASC/LSC.

I understand this is necessary for the test, but are we also planning to merge it in main?

Reply:
Can you move lines - https://github.com/facebookresearch/spot-sim2real/pull/76/files#diff-a5b8eac17230285c05f88335ad5821aa16beb1580516a946bafdf0e2475ec436R429-R430 above arg parse?

So that way you will have the bounding box detector type ready. We can then use that info to create `--bounding-box-detector` with required algo.